server/mailer/BaseMailer.js

```diff
@@ -22,6 +22,11 @@ export default class BaseMailer {
               user: process.env.SMTP_USER,
               pass: process.env.SMTP_PASS,
             },
+            tls: {
+              // to not fail on invalid certs:
+              // set SMTP_REJECT_UNAUTHORIZED=false in .env file
+              rejectUnauthorized: process.env.SMTP_REJECT_UNAUTHORIZED == 'true',
```

Comment:
LGTM. I would keep the "loose" equality you currently have vs. what the GH commend said because it will coerce boolean to string in the event the env var parses as a boolean value.